### ISSUE TITLE
Minecraft using integrated GPU instead of dedicated GPU for modded installations

### ISSUE THREAD

**Issue by Eatham532** [modrinth/code#303]
### Describe the bug

The integrated GPU is being used instead of the dedicated GPU for modded installations. Vanilla Minecraft uses the appropriate GPU.

### Steps to reproduce

1, Install a modpack
2. Create a world
3. Press F3 and notice the results. 

### Expected behavior

Minecraft should use the dedicated GPU.

**Comment by cootshk** [comment]
This is windows being s****y, go into power saving settings, find javaw.exe, and set it to high performance

**Comment by cootshk** [comment]
[https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwiT9oDi9fP_AhU_GVkFHeDrDjYQFnoECA8QAw&url=https%3A%2F%2Fwww.intel.com%2Fcontent%2Fwww%2Fus%2Fen%2Fsupport%2Farticles%2F000090168%2Fgraphics.html&usg=AOvVaw00gsLWM-4LAqQhMJ46X2E8&opi=89978449](tutorial)

**Comment by Felix14-v2** [comment]
> This is windows being s****y

The launcher may still have the functionality to detect a high-performance graphics card and run the game with it. At least this is implemented in Legacy Launcher.

**Comment by HyperSoop** [comment]
yep, the launcher should be able to get the point across to the OS
